src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,22 @@
+import { faGithubAlt } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
We should be using tsx here not jsx

Reply:
My mistake! Still getting used to the extensions 😅